.gitignore

```
@@ -20,3 +20,4 @@ react-native
 
 solidity/cache
 solidity/build
+*.swp
```

Youve added a couple of `.DS_Store` files in this PR
```suggestion
*.swp
.DS_Store
```